### PR TITLE
Fix getting status for OS-provisioner

### PR DESCRIPTION
**Pull request by katyafervent**

### CODE REVIEW

File: kqueen/engines/openstack_kubespray.py [hunk 1]
```diff
@@ -822,7 +824,7 @@ def shrink(self, *, resources, remove_hostnames):
         return resources
 
     def _get_userdata(self):
-        docker_bip = {"bip": self.meta["docker_bip_network"]}
+        docker_bip = {"bip": self.cluster.metadata.get("docker_bip_network", "10.13.0.1/16")}
```
Comment (naumvd95) [inline kqueen/engines/openstack_kubespray.py]: if u want to define "10.13.0.1/16" as sane default, please define it in kqueen config base.py

File: kqueen/engines/openstack_kubespray.py [hunk 2]
```diff
@@ -756,6 +754,10 @@ def deprovision(self, volume_names):
                         self.c.remove_router_interface(router, port_id=i.id)
                         time.sleep(2)
                     self.c.delete_router(router.id)
+                    logger.debug("Router {0} has been deleted".format(self.stack_name))
+                if "resources" not in self.cluster.metadata:
+                    # Deployment failed before instance spawning
```
Comment (naumvd95) [inline kqueen/engines/openstack_kubespray.py]: logging? exception?

Comment (katyafervent) [reply kqueen/engines/openstack_kubespray.py]: exception was raised while provisioning, in that case we are handling deprovision of failed deployment

Comment (naumvd95) [reply kqueen/engines/openstack_kubespray.py]: ok, agree

**Pull request merged**